### PR TITLE
Add support for ramsey/uuid>=4.1

### PR DESCRIPTION
**Pull request by Toilal**
#14

### CODE REVIEW

File: src/Ekreative/UuidExtraBundle/Serializer/UuidNormalizer.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ public function denormalize($data, $class, $format = null, array $context = [])
 
     public function supportsDenormalization($data, $type, $format = null)
     {
-        return $type === UuidInterface::class || $type === Uuid::class;
+        return $type === UuidInterface::class || $type === Uuid::class || $type === "Ramsey\Uuid\Lazy\LazyUuidFromString";
```
Comment (mcfedr) [inline src/Ekreative/UuidExtraBundle/Serializer/UuidNormalizer.php:26]: you can use a `::class` here, its not a problem if the class doesnt exist.

Comment (Toilal) [reply src/Ekreative/UuidExtraBundle/Serializer/UuidNormalizer.php:26]: This would break for users that still run `ramsey/uuid<4.1`.

Comment (Toilal) [reply src/Ekreative/UuidExtraBundle/Serializer/UuidNormalizer.php]: Learned something : PHP may be the single langage where importing a missing class doesn't fail :p.

Tried unit tests with `ramsey/uuid:^3`, it works too.

**Comment by mcfedr** [comment]
Thanks, fixes #14

**Comment by mcfedr** [comment]
Released 4.1.0

**Pull request merged**